stdlib/public/core/ExistentialCollection.swift.gyb

```diff
@@ -164,6 +195,51 @@ internal class _${Kind}Box<S : ${Kind}Type>
   override func _underestimateCount() -> Int {
     return _base.underestimateCount()
   }
+  
+  override func _map<T>(
+    @noescape transform: Element throws -> T
+  ) rethrows -> [T] { 
+    return try _base.map(transform)
+  }
+  
+  override func _filter(
+    @noescape includeElement: (S.Generator.Element) throws -> Bool
+  ) rethrows -> [S.Generator.Element] {
+     return try _base.filter(includeElement)
+  }
+  
+  override func _forEach(
+    @noescape body: S.Generator.Element throws -> Void
+  ) rethrows {
+    return try _base.forEach(body)
+  }
+  
+  override func _dropFirst(n: Int) -> AnySequence<Element> {
+    return _base.dropFirst(n)
```

Comment:
I don't think this actually forwards to the underlying sequence.  It just forces the overload resolution to pick up the default implementation.